releasev1.5.0 (#383)
fluent/fluent-operator: CHANGELOG.md

## 1.5.0 / 2022-09-23

### Features

> - Accomodate securityContext (#344)

Reviewer: ```suggestion
- Add SecurityContext to FluentBit CRD (#344)
```

> - Support OpenTelemetry and Node Exporter Metrics plugins (#345)

Reviewer: ```suggestion
- Add OpenTelemetry output plugin (#345)
- Add Node Exporter Metrics input plugin (#345)
```

> - Support fluentd loki output (#346)

Reviewer: ```suggestion
- Add Fluentd Loki output plugin (#346)
```

> - Support prometheus scrape metrics input plugin and add the PrometheusRemoteWrite output plugin (#362)

Reviewer: ```suggestion
- Add Prometheus scrape metrics input plugin (#362)
- Add Prometheus remote write output plugin (#362)
```

> - Support fluentbit metrics input plugin (#366)

Reviewer: ```suggestion
- Add Fluent Bit metrics input plugin (#366)
```

> - Add alias for filter (#370)

Reviewer: ```suggestion
- Add alias for the filter plugin (#370)
```

> - Support customplugin (#377)

Reviewer: ```suggestion
- Support custom plugins (#377, #380)
```

> - Receive non-kubernetes format log (#382)

Reviewer: ```suggestion
- Support receiving Non-K8s format log by Fluentd (#382)
- Add HostNetwork support for the Fluent Bit DaemonSet (#369)
```

### ENHANCEMENT

> - Simplify the step of issue report (#334)

Reviewer: ```suggestion
- Add node label to the Prometheus remote write metrics(#372)
- Simplify the steps of the issue report (#334)
```

> - Add fluentd loki output documentation (#349)

Reviewer: ```suggestion
- Add Fluentd Loki output plugin docs (#349)
```

- Add guide for node exporter metrics plugin (#353)

> - Docs: update the index of plugins for fluent-bit (#354)

Reviewer: ```suggestion
- Docs: update the index of the Fluent Bit plugins (#354)
```

- Add release drafter (#379)

> - Perf: optimize code (#380)

Reviewer: ```suggestion
```

> - Add documents for prometheus-scrape-metrics input plugin and prometheus-remote-write output plugin (#381)

Reviewer: ```suggestion
- Add docs for the Prometheus scrape metrics input plugin and the Prometheus remote write output plugin (#381)
```

### BUGFIX 
- Fix the bug of feature request issue will be closed by mistake. (#341) 
- Correct invalid links (#347)

> - Fix: Fluent-bit supports the hostNetwork configuration of pod (#369)

Reviewer: ```suggestion
```

> - Fix: Add node label to the configuration YAML file for prometheusremoteWrite (#372)

Reviewer: ```suggestion
```
